packages/obuilder/obuilder.0.4/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+synopsis: "Run build scripts for CI"
+description:
+  "OBuilder takes a build script (similar to a Dockerfile) and performs the steps in it in a sandboxed environment."
+maintainer: ["talex5@gmail.com"]
+authors: ["talex5@gmail.com"]
+homepage: "https://github.com/ocurrent/obuilder"
+doc: "https://ocurrent.github.io/obuilder/"
+bug-reports: "https://github.com/ocurrent/obuilder/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "lwt" {>= "5.5.0"}
+  "astring"
+  "fmt" {>= "0.8.9"}
+  "logs"
+  "cmdliner" {>= "1.1.0"}
+  "tar-unix" {>= "2.0.0"}
+  "yojson"
```

Comment:
```suggestion
  "yojson" {>= "1.6.0"}
```
For [(failed: Unbound type constructor Yojson.Safe.t)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/7fe26856e905c172d537f3d35a9390755d8a9fbd/variant/opam-2.1,compilers,4.14,obuilder.0.4,lower-bounds)